src/components/solution-control/solution-control.tsx

```diff
@@ -88,6 +87,7 @@ export const SolutionControl = ({
         onApply={onApply}
         onModify={onModify}
         onDelete={solutionName => openDeleteDialog({ solutionName })}
+        isModifyDisabled={selected === selectedProp}
```

Comment:
it should be disabled not when it's selected but when no changes have been made to the preset

Reply:
Also, there is `_presetModifiedState` in the filterStore that could be useful

Reply:
If we have no changes in preset selectedProp(it is a preset that is showing by UI) will be equal to selected preset that is chosen by user in list. If  any change heve been done in our preset selectedProp will be undefined until we save preset or modify some preset in list.

Reply:
Second point: I'd show a confirm message when modifying a preset, because now user can simply ruin their settings by a single click. We should avoid it.

Reply:
Added